json/src/spec/spec.rs

```diff
@@ -24,17 +24,29 @@ use serde_json::Error;
 /// Fork spec definition
 #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Deserialize)]
 pub enum ForkSpec {
-	EIP150,
-	EIP158,
```

Comment:
`ForkSpec` derives `Ord`, I don't know if changing the order here breaks something or not?

Reply:
reverted